tests/raftstore/util.rs

```diff
@@ -128,8 +127,18 @@ pub fn new_server_config(cluster_id: u64) -> ServerConfig {
 
 pub fn new_readpool_cfg() -> ReadPoolConfig {
     ReadPoolConfig {
-        storage: ReadPoolInstanceConfig::default_for_test(),
-        coprocessor: ReadPoolInstanceConfig::default_for_test(),
+        storage: StorageReadPoolConfig {
```

Comment:
`StorageReadPoolConfig::with_concurrency(1)` ?

Reply:
it is not available. There is `readpool::Config::default_with_concurrency`, but not `StorageReadPoolConfig::default_with_concurrency`.